source/extensions/stat_sinks/hystrix/hystrix.cc

```diff
@@ -43,6 +45,18 @@ void ClusterStatsCache::printRollingWindow(absl::string_view name, RollingWindow
   out_str << std::endl;
 }
 
+void HystrixSink::addHistogramToStream(const QuantileLatencyMap& latency_map, absl::string_view key,
+                                       std::stringstream& ss) {
+  ss << ", \"" << key << "\": {";
+  bool is_first = true;
```

Comment:
This seems a place where a join operation would make sense, but I can see that there is already a stylized approach to building these strings in this code.

Reply:
Do you want me to consider changing the code to use join?

Reply:
Up to you, TODO is fine as well.